package.json

```diff
@@ -52,7 +52,9 @@
 	},
 	"dependencies": {},
 	"devDependencies": {
+		"cldr-data": ">=26",
 		"commitplease": "2.0.0",
+		"globalize-compiler": "0.1.1",
```

Comment:
Globalize compiler is still too young, so it's hard to tell. Surely, no breaking changes are expected for patch releases and I hope minors don't too.